solution.js

```diff
@@ -1,5 +1,111 @@
 const { nums, words } = require("./data/data.js");
 
+class Node {
+  constructor(data) {
+    this.data = data;
+    this.next = null;
+  }
+}
+
+class LinkedList {
+  constructor(head = null) {
+    this.head = head;
+  }
+
+  insert(value) {
+    let newNode = new Node(value);
+    newNode.next = this.head;
+    this.head = newNode;
+  }
+
+  size() {
+    let count = 0;
+    let currentNode = this.head;
+    while (currentNode) {
+      count++;
+      currentNode = currentNode.next;
+    }
+    return count;
+  }
+
+  delete(position) {
+    let count = 1;
+    let currentNode = this.head;
+    while (count < position - 1) {
+      count++;
+      currentNode = currentNode.next;
+    }
+    currentNode.next = currentNode.next.next;
+  }
+
+  getFirst() {
+    return this.head ? this.head : null
```

Comment:
Could also be written using short circuiting
```suggestion
    return this.head || null
```